packages/ketcher-polymer-editor-react/src/components/monomerLibrary/RnaBuilder/RnaEditor/RnaEditor.tsx

```diff
@@ -65,25 +65,18 @@ export const RnaEditor = ({ duplicatePreset, activateEditMode }) => {
     dispatch(setIsEditModeAction(value));
   };
 
-  const [expanded, setExpanded] = useState(false);
+  const [expanded, setExpanded] = useState(!!activePreset?.name || isEditMode);
 
   useEffect(() => {
-    if (activePreset) {
-      if (activePreset.name || isEditMode) setExpanded(true);
-      return;
-    }
-
-    if (hasPresets) {
-      dispatch(setActivePreset(presets[0]));
-    } else {
+    if (!hasPresets) {
       dispatch(createNewPreset());
       dispatch(setActiveRnaBuilderItem(RnaBuilderPresetsItem.Presets));
     }
   }, [activePreset]);
 
   const expandEditor = () => {
     setExpanded(!expanded);
-    if (!activePreset.presetInList) {
+    if (!activePreset?.presetInList) {
```

Comment:
Could you, please, also check the behaviour for editing presets?
1. Just create any preset
2. Click edit
3. Try to change a name of preset
4. Focus from input field is constantly lost after single character typing.